src/browser_manager.ts

```
@@ -155,6 +182,16 @@
           //downloadsPath: downloadsPath,
         });
+      } else if (channel) {
+        {
+          this.browser = await chromium.launch({
+            headless: headless,
+            timeout: 0,
+            args: ["--ignore-https-errors", "--ignore-certificate-errors"],
+            channel: channel,
+            //downloadsPath: downloadsPath,
+          });
+        }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)